mephisto/client/cli_commands.py

```diff
@@ -0,0 +1,184 @@
+from mephisto.utils.rich import console, print_out_valid_options, create_table
+import rich_click as click  # type: ignore
+from rich import print
+
+
+def get_wut_arguments(args):
+    """Display information about hydra config properties"""
+    from mephisto.operations.registry import (
+        get_blueprint_from_type,
+        get_crowd_provider_from_type,
+        get_architect_from_type,
+        get_valid_blueprint_types,
+        get_valid_provider_types,
+        get_valid_architect_types,
+    )
+
+    if len(args) == 0:
+        print(
+            "\n[red]Usage: mephisto wut <abstraction>[=<type>] [...specific args to check][/red]"
+        )
+        abstractions_table = create_table(
+            ["Abstraction", "Description"], "\n\n[b]Abstractions[/b]"
+        )
+        abstractions_table.add_row(
+            "blueprint",
+            f"The blueprint determines the task content. \nValid blueprints types are [b]{get_valid_blueprint_types()}[/b]",
+        )
+        abstractions_table.add_row(
+            "architect",
+            f"The architect determines the server where a task is hosted. \nValid architects types are [b]{get_valid_architect_types()}[/b]",
+        )
+        abstractions_table.add_row(
+            "requester",
+            f"The requester is an account for a crowd provider. \nValid requester types types are [b]{get_valid_provider_types()}[/b]. \n"
+            "\nUse `mephisto requesters` to see registered requesters, and `mephisto register <requester type>` to register.",
+        )
+        abstractions_table.add_row(
+            "provider",
+            f"The crowd provider determines the source of the crowd workers. \nValid provider types are [b]{get_valid_provider_types()}[/b]",
+        )
+        console.print(abstractions_table)
```

Comment:
stray console statement?

Reply:
I guess console.print and print are different yet similar?

Reply:
console.print prints content to your terminal. So this prints out your abstractions_table when you type `mephisto wut`